src/presentational/Components/Common/CountdownTimer/CoundownTimer.js

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+
+import Box from '@material-ui/core/Box';
+import Typography from '@material-ui/core/Typography';
+
+import styles from "./CountdownTimer.module.scss";
+
+function CoundownTimer() {
+
+  const difference = new Date("2020-01-11") - new Date();
+
+  const [timeLeft, setTimeLeft] = useState({
+    days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+    hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+    minutes: Math.floor((difference / 1000 / 60) % 60),
+    seconds: Math.floor((difference / 1000) % 60)
+  });
+
+  function tickTock() {
+    let currTimeLeft = {};
+    if (difference > 0) {
+      currTimeLeft = {
+        days: Math.floor(difference / (1000 * 60 * 60 * 24)),
+        hours: Math.floor((difference / (1000 * 60 * 60)) % 24),
+        minutes: Math.floor((difference / 1000 / 60) % 60),
+        seconds: Math.floor((difference / 1000) % 60)
+      };
```

Comment:
We can just keep `difference` in state, do all the calculation in JSX section; so that you do not have to repeat the formula.